src/downshift.js

```diff
@@ -562,15 +562,27 @@ class Downshift extends Component {
 
   button_handleClick = event => {
     event.preventDefault()
+
+    if (
+      /* istanbul ignore next (react-native) */
```

Comment:
I think you may need to lift this `instanbul` declarative one line higher to ensure it's ignored for now.